flask_peewee/tests/test_rest.py

```diff
@@ -631,7 +635,7 @@ def test_filtering(self):
 
         # do a filter with an IN operator and multiple IDs
         # https://github.com/coleifer/flask-peewee/issues/112
-        resp = self.app.get('/api/note?id__in=1,2,5')
+        resp = self.app.get('/api/note?id__in=1&id__in=2&id__in=5')
```

Comment:
Nice! Has this always been supported or is this a hint of a potentially breaking change?